connect-examples/oauth/java/src/main/java/com/squareup/oauthexample/OAuthHandler.java

```diff
@@ -128,8 +132,9 @@ public void handle(HttpExchange t) throws IOException {
       }
 
       SquareClient client = new SquareClient.Builder()
-            .environment(ENVIRONMENT)
-            .build();
+          .environment(ENVIRONMENT)
+          .userAgentDetail("sample_app_oauth_java")
```

Comment:
Could you add a comment explaining that this should be replaced or removed when building on top of this sample?

Reply:
Same for all other instances of `user agent details`